internal/util/paramtable/basetable.go

```diff
@@ -128,7 +127,6 @@ func (gp *BaseTable) loadFromCommonYaml() bool {
 		}
 		return true
 	}
-	log.Debug("failed to find common.yaml in config, skip..")
```

Comment:
common.yaml has been deleted，is this function work?

Reply:
This won't work, just in case user with old configs put some of the configs into common.yaml